docs/content/api/errors.en.md

```diff
@@ -60,4 +60,7 @@ See the [error protocol buffers](https://github.com/trisacrypto/trisa/blob/main/
 | 152 | UNPARSEABLE_TRANSACTION | Could not parse the transaction data record; specify the type in extra. |
 | 153 | MISSING_FIELDS | There are missing required fields in the transaction data, a list of these fields is specified in extra. |
 | 154 | INCOMPLETE_IDENTITY | The identity record is not complete enough for compliance purposes of the receiving VASPs. Required fields or format specified in extra. |
-| 155 | VALIDATION_ERROR | There was an error validating a field in the transaction data (specified in extra). |
\ No newline at end of file
+| 155 | VALIDATION_ERROR | There was an error validating a field in the transaction data (specified in extra). |
+| 198 | COMPLIANCE_PERIOD_EXCEEDED | Send by originating party if the review period has exceeded the required compliance timeline. |
```

Comment:
Should these errors also be added as options in the `reject transaction` modal?

Reply:
Yes please!